packages/appium/lib/extension/manifest-migrations.js

```diff
@@ -0,0 +1,94 @@
+import log from '../logger';
+
+/**
+ * This contains logic to migrate old versions of `extensions.yaml` to new versions.
+ *
+ * @module
+ */
+
+/**
+ * Collection of functions to migrate from one version to another
+ * @type {{[P in keyof ManifestDataVersions]?: Migration}}
+ */
+const Migrations = {
+  /**
+   * If `installPath` is missing from the `ExtManifest` for any extension, return `true`.
+   *
+   * We cannot easily determine the path of any given extension here, but returning `true`
+   * will cause the `Manifest` to sync with the filesystem, which will populate the missing field.
+   *
+   * @type {Migration}
+   */
+  3: (data) => {
```

Comment:
shall we use the constant instead?

Reply:
There is no constant for this...

There's `CURRENT_SCHEMA_REV = 3`, but that is supposed to be updated when we increment the revision, so we cannot use it.

However: It's probably a good idea to define one anyway, so I'll do that.

Reply:
OK, this is done and I added a note in the `README` about it